conf/ocsci/production-aws-ipi.yaml

```diff
@@ -3,3 +3,4 @@ ENV_DATA:
   platform: 'aws'
   deployment_type: 'ipi'
   worker_instance_type: 'm5.4xlarge'
+  allow_lower_instance_requirements: false
```

Comment:
@andrewschoen @zmc hello, can you please make sure you are passing this configuration file in your execution or adding this one parameter? 
`allow_lower_instance_requirements: false`

I am for first option as then you will use the production config file as we do and no need to adding other parameters when we add some later.

Reply:
@petr-balogh I've opened https://github.com/red-hat-storage/ocs-ci/pull/871 for this. Thanks!

Reply:
@andrewschoen thanks!